trellis-triplestore/src/main/java/org/trellisldp/triplestore/TriplestoreResourceService.java

```diff
@@ -261,11 +284,14 @@ private void emitEvents(final IRI identifier, final Literal time, final Dataset
      *   }
      * }
      * </code></pre></p>
+     *
+     * <p>Note: this query does not retrieve the user-managed rdf:type values
+     * nor does it retrieve any ldp:inbox value.
      */
-    private void emitEventsForAdjacentResources(final EventService svc, final IRI parent,
-            final Literal time, final Optional<String> baseUrl, final Dataset dataset) {
-        final Boolean isDelete = dataset.contains(of(PreferAudit), null, RDF.type, AS.Delete);
-        final Boolean isCreate = dataset.contains(of(PreferAudit), null, RDF.type, AS.Create);
+    private void emitEventsForAdjacentResources(final EventService svc, final IRI parent, final Session session,
+            final OperationType opType, final Literal time) {
+        final Boolean isDelete = opType == OperationType.DELETE;
```

Comment:
Just a note for the future-- this is a smell. We'll want to introduce some polymorphism at some point to avoid this kind of switchery. But I'm not asking for that now.